functions/Get-SqlMaxMemory.ps1

```diff
@@ -1,14 +1,13 @@
-Function Get-SqlMaxMemory
+Function Get-Sqlmaxmemory
```

Comment:
Please do keep the camel case caps. I saw your recent comment, we were consolidating issues from Trello and there are sorta duplicates. Please do rename to Get-DbaMaxMemory, Set-DbaMaxMemory and Test-DbaMaxMemory. I will take care of the aliasing in the psm1 and psd1.